<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplitterPane</class>
 <widget class="QWidget" name="SplitterPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>1216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelTitle">
          <property name="text">
           <string>Pane</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonArrange">
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Split this pane vertically</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="list-remove"/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSplitVertical">
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Split this pane vertically</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::GoDown"/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSplitHorizontal">
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Split this pane horizontally</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::GoNext"/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonClose">
          <property name="maximumSize">
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Close this pane</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="window-close"/>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidgetContent">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageMenu">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>545</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelSelectContent">
            <property name="text">
             <string>Select content</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonMenu">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string>Menu</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::WindowNew"/>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>544</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageContent"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>SplitterPane</receiver>
   <slot>closePane()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>837</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSplitHorizontal</sender>
   <signal>clicked()</signal>
   <receiver>SplitterPane</receiver>
   <slot>splitPaneHorizontally()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSplitVertical</sender>
   <signal>clicked()</signal>
   <receiver>SplitterPane</receiver>
   <slot>splitPaneVertically()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonArrange</sender>
   <signal>clicked()</signal>
   <receiver>SplitterPane</receiver>
   <slot>arrangeSplitter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1156</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMenu</sender>
   <signal>clicked()</signal>
   <receiver>SplitterPane</receiver>
   <slot>showMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>629</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>607</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>closePane()</slot>
  <slot>splitPaneVertically()</slot>
  <slot>splitPaneHorizontally()</slot>
  <slot>arrangeSplitter()</slot>
  <slot>showMenu()</slot>
 </slots>
</ui>
